app/models/status.rb

```diff
@@ -198,10 +198,20 @@ def preview_card
   end
 
   def title
-    if destroyed?
-      "#{account.acct} deleted status"
+    return "#{account.acct} deleted status" if destroyed?
+
+    preview = proper.spoiler_text.presence || proper.text
+    if preview.length > 30 || preview[0, 30].include?("\n")
+      preview = preview[0, 30]
+      preview = preview[0, preview.index("\n").presence || 30] + '…'
```

Comment:
The fact that we're re-implementing ActiveSupport text helpers in here makes me think maybe this method has overstayed its welcome here and maybe it belongs in some presentation layer.

Reply:
Yeah, I thought about moving it to the RSS serializers at first but forgot it was used by the OEmbed endpoint. I'll think about where to move it (maybe the Formatter)

Reply:
I'm not sure why the OEmbed endpoint needs titles, to be honest. Twitter's doesn't give tweets titles. We could get rid of it.

Reply:
Does twitter use OEmbed?

Reply:
Yeah!